api/v1alpha1/servicedefaults_types.go

```diff
@@ -144,3 +169,23 @@ func (e ExposeConfig) toConsul() capi.ExposeConfig {
 		Paths:  paths,
 	}
 }
+
+func (e ExposeConfig) validate() []*field.Error {
+	var errs field.ErrorList
+	for i, path := range e.Paths {
+		if path.Path != "" && !strings.HasPrefix(path.Path, "/") {
+			errs = append(errs, field.Invalid(field.NewPath("spec").Child("expose").Child(fmt.Sprintf("paths[%d]", i)).Child("path"), path.Path, `must begin with a '/'`))
+		}
+		switch path.Protocol {
+		case "http", "http2":
```

Comment:
Can we use something like `var allowPathProtocols = []string{"http", "http2"}` and use it for this case and constructing the error message?